_drafts/2018-03-01-q2-goals.md

```diff
@@ -0,0 +1,77 @@
+---
+
+title: "Product goals for 2018 Q2"
+author: [huszerlbence]
+image: "../img/posts/q2_goals.jpg"
+tags: [goals, community, roadmap, cloud, Azure]
+
+---
+
+In Q2 there are some very interesting plans on our roadmap like SN6 to SN7 upgrade patch, Cloud compatibility developments to prepare sensenet for Azure and later to other cloud based environments and a huge UX revamp project to name a few. Online document editing is also a requested feature so we are working to implement Office Web Apps Server to sensenet in the coming months. Read our article to get some updates about plans and goals for Q2 2018.
+
+---
+
+### SN6 to SN7 - Patch is coming
+
+As we no longer provide updates for older versions of sensenet, it is crucial to upgrade every project to the latest version. We are working on a patch which can upgrade existing projects from version 6 to the latest release. Once upgraded to SN7 you can easily install future patches for this version and keep your solution up to date.
+
+We plan to publish this upgrade patch in Q2 this year. Stay tuned as we'll share more info about development process in the coming weeks.
+
+### Cloud compatibility
+![Cloud compatibility](/img/posts/cloud_compatibility.jpg "sensenet goes to cloud")
+The next milestone in the lifetime of SN7 is cloud compatibility which involves lot of additional researches and developments. 
+
+Read below some challenges we will be working on in Q2:
+
+#### Azure SQL & AD
```

Comment:
Not sure this will happen in the short term, I think we should remove this section for now.